Games/types/Mafia/const/Priority.js

```diff
@@ -132,6 +136,7 @@ module.exports = {
 	PRIORITY_MODIFY_INVESTIGATIVE_RESULT_DEFAULT,
 	PRIORITY_MAKE_INNOCENT,
 	PRIORITY_REVEAL_TARGET_ON_DEATH,
+	PRIORITY_DATE,
```

Comment:
I put it with the investigative roles because it also kind of acts like an alignment check and thought it should have the max priority of all its features